index.css

```diff
@@ -0,0 +1,79 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: #f7f7f9;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
```

Comment:
- [OPTIONAL] Kindly note that the general background color of the headline section is `ffffff` as used in the Figma design. Kindly adjust that